src/Elliptic/Executables/Elasticity/SolveElasticity.hpp

```diff
@@ -163,10 +164,17 @@ struct Metavariables {
                                          typename system::primal_fluxes>>;
 
   using analytic_solution_fields = typename system::primal_fields;
-  using observe_fields = tmpl::append<
-      analytic_solution_fields,
-      tmpl::list<Elasticity::Tags::Strain<volume_dim>,
-                 Elasticity::Tags::PotentialEnergyDensity<volume_dim>>>;
+  using error_compute = ::Tags::ErrorsCompute<analytic_solution_fields>;
+  using error_tags = db::wrap_tags_in<Tags::Error, analytic_solution_fields>;
+  using observe_fields = tmpl::push_back<
+      tmpl::append<
+          analytic_solution_fields, error_tags,
+          tmpl::list<Elasticity::Tags::Strain<volume_dim>,
+                     Elasticity::Tags::PotentialEnergyDensity<volume_dim>>>,
+      domain::Tags::Coordinates<volume_dim, Frame::Inertial>>;
+  using non_tensor_compute_tags =
```

Comment:
(optional) I think `non_tensor_compute_tags` -> `observer_compute_tags` would be clearer in the metavariables

Reply:
I'm going to avoid a rename in this PR since it could make rebasing much more annoying. I have no objection to the suggested rename, though.

Reply:
Ok